### PR TITLE
New metaquery api

### PR DESCRIPTION
**Pull request by CaseyLeask**
Currently a work in progress

### CODE REVIEW

File: example/metaquery-images.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    if ( !matches || !matches.length ) { return; }
+
+    var elements = document.getElementsByTagName( 'img' );
+
+    for ( var i = 0; i < elements.length; i++ ) {
+      var el = elements[i],
+        template = el.getAttribute( 'data-mq-src' );
+
+      if ( template ) {
+        el.src = template.replace( '[breakpoint]', matches.join('-') );
```
Comment (benschwarz) [inline example/metaquery-images.js:14]: What order will the matches be returned in? Order of `<meta>` source order appearance? Hmmm…

Comment (CaseyLeask) [inline example/metaquery-images.js:14]: I believe so? It's the same as the old version, but the answer might differ between browsers.

Currently going through https://developer.mozilla.org/en-US/docs/Web/API/document.getElementsByTagName and seeing if I can work it out.

Comment (benschwarz) [inline example/metaquery-images.js:14]: Sounds reasonable…

**Comment by benschwarz** [comment]
Hmm. This is just a collection of the other PRs, right?

**Comment by CaseyLeask** [comment]
Nope. I've rebased the PR to show the changes a little more clearly.

At the moment the changes are
- Using ES5 to simplify the code a little
- Adding metaquery-images.js to the minify process
- Some minor refactoring

**Pull request merged**